bom/pom.xml

```diff
@@ -71,7 +71,7 @@
       <dependency>
         <groupId>com.datastax.oss</groupId>
         <artifactId>native-protocol</artifactId>
-        <version>1.5.0</version>
+        <version>1.5.1-SNAPSHOT</version>
```

Comment:
Just a placeholder for now to use a locally-built version of native-protocol that includes https://github.com/datastax/native-protocol/pull/42.  To be replaced by an actual version once we release a new version of native-protocol.

Reply:
native-protocol 1.5.1 has now been released